packages/cli/src/credentials/credentials.service.ee.ts

```diff
@@ -25,6 +25,36 @@ export class EECredentialsService extends CredentialsService {
 		return { ownsCredential: true, credential };
 	}
 
+	/**
+	 * Retrieve the sharing that matches a user and a credential.
+	 */
+	static async getSharing(
+		user: User,
+		credentialId: number | string,
+		relations: string[] = ['credentials'],
```

Comment:
Nit: No relations requires an empty array, not ideal, perhaps consolidated as an option with the next param. (Comes from before, feel free to disregard.)